app/views/organizations/adopter_fosterer/adopted_pets/_pet_files.html.erb

```diff
@@ -0,0 +1,36 @@
+<div id="pet_files">  
```

Comment:
This table is almost identical to the one we use on staff/pets to upload and show files, right? `app/views/organizations/staff/pets/_file_attachment_table.html.erb` The main difference is whether or not we want to render a delete button.

I am wondering if instead of duplicating a lof ot this logic here, if we can move this partial `app/views/organizations/staff/pets/_file_attachment_table.html.erb` to a shared folder e.g. at `aap/views/organizations/shared` and then conditionally render the delete button based on the User type, e.g. if the user role is staff we render the delete button. Does that make sense?

Reply:
Thanks for the input. Common file does make sense and easy to maintain. Have updated changes for this.